pkg/webhook/policy.go

```diff
@@ -118,6 +131,15 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 		return vResp
 	}
 
+	var requestResponse requestResponse
```

Comment:
nit: is an empty string safe to provide to ReportRequest? This could happen if a new return is added.

Reply:
ping